satinfo/amsua_n15/readme

```diff
@@ -4,5 +4,5 @@ n15  19990101 060000  19990101 060000  amsua  0							# off due to noisy (as in
 n15  19990101 120000  20001029 240000  amsua 15  1 2 3 4 5 6 7 8 9 10 11 12 13 14 15   		# channel 11 off (as in CFSR; Merra turns it off 20020401)
```

Comment:
still need to remove "11" from this line and set number of assimilated channels to 14.

Reply:
moved comment on Ch 11 to next line (Ch11 turned off on 20000130)